<a href="https://colab.research.google.com/github/Vlad-Scooby/Estudo.ipynb/blob/main/PythonBrasil(Estruturas_de_Decis%C3%A3o).ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

#Desafio DevPro: Aluno [Vlad Scooby](https://github.com/Vlad-Scooby)

#Exercicios [Python Brasil](https://wiki.python.org.br/ListaDeExercicios)

#[Estrutura de Decisão](https://wiki.python.org.br/EstruturaDeDecisao)


## Exercício 1

### Faça um Programa que peça dois números e imprima o maior deles.

In [ ]:
print('<<!Escolha do número maior!>>')
print()

numero1 = int(input('Digite um número: '))
numero2 = int(input('Digite outro número: '))
print()

if numero1 > numero2:
  print(f'O número {numero1} é maior que o número {numero2}')

elif numero1 == numero2:
  print(f'O número {numero1} é igual ao número {numero2}')

else:
  print(f'O número {numero2} é maior que o número {numero1}')

<<!Escolha do número maior!>>

Digite um número: 111
Digite outro número: 67

O número 111 é maior que o número 67


## Exercício 2

### Faça um Programa que peça um valor e mostre na tela se o valor é positivo ou negativo.

In [ ]:
valor = int(input('Digite um número: '))
print()

if valor <= -1:
  print(f'O número {valor} é um valor negativo')

else:
  print(f'O número {valor} é um valor positivo')

Digite um número: -34

O número -34 é um valor negativo


## Exercício 3

### Faça um Programa que verifique se uma letra digitada é "F" ou "M". Conforme a letra escrever: F - Feminino, M - Masculino, Sexo Inválido.

In [ ]:
print('Escolha do Sexo')
print('Digite "F" para feminino, "M" para masculino, "I" para indefinido')
print()
qual_sexo = input('Digite qual seu sexo: ').upper()
print()

if qual_sexo == 'M' or qual_sexo == 'MASCULINO':
  print(f'Sexo: Masculino')

elif qual_sexo == 'F' or qual_sexo == 'FEMININO':
  print(f'Sexo: Feminino')

elif qual_sexo == 'I' or qual_sexo == 'INDEFINIDO':
  print(f'Sexo: Indefinido')

else:
  print(f'Sexo ínvalido, repita a operação')

Escolha do Sexo
Digite "F" para feminino, "M" para masculino, "I" para indefinido

Digite qual seu sexo: ghghghgh

Sexo ínvalido, repita a operação


## Exercício 4

### Faça um Programa que verifique se uma letra digitada é vogal ou consoante.

In [ ]:
vogal_ou_consoante = input('Digite uma Letra: ').upper()
print()

if vogal_ou_consoante == 'A' or vogal_ou_consoante == 'E' or vogal_ou_consoante == 'I' or vogal_ou_consoante == 'O' or vogal_ou_consoante == 'U':
  print(f'A letra: {vogal_ou_consoante} é uma Vogal.')

else:
  print(f'A letra: {vogal_ou_consoante} é uma Consoante.')

Digite uma Letra: g

A letra G é uma Consoante.


## Exercício 5

### Faça um programa para a leitura de duas notas parciais de um aluno. O programa deve calcular a média alcançada por aluno e apresentar:
###A mensagem "Aprovado", se a média alcançada for maior ou igual a sete;
###A mensagem "Reprovado", se a média for menor do que sete;
###A mensagem "Aprovado com Distinção", se a média for igual a dez.

In [ ]:
primeira_nota = float(input('Digite a primeira nota: '))
segunda_nota = float(input('Digite a segunda nota:'))
print()

media_das_notas = (primeira_nota + segunda_nota) / 2

if media_das_notas >= 10:
  print(f'Nota do Aluno: {media_das_notas}')
  print(f'Aluno Aprovado com Destinção')

elif media_das_notas >= 7:
   print(f'Nota do Aluno: {media_das_notas}')
   print(f'Aluno Aprovado') 
  
elif media_das_notas >= 6:
   print(f'Nota do Aluno: {media_das_notas}')
   print(f'Aluno está de Recuperação')

else:
   print(f'Nota do Aluno: {media_das_notas}')
   print(f'Aluno Reprovado')


Digite a primeira nota: 6.1
Digite a segunda nota:6

Nota do Aluno: 6.05
Aluno está de Recuperação


## Exercício 6

### Faça um Programa que leia três números e mostre o maior deles.

In [ ]:
primeiro_numero = int(input('Digite um número: '))
segundo_numero = int(input('Outro número: '))
terceiro_numero = int(input('Mais um número: '))
print()

if primeiro_numero > segundo_numero and primeiro_numero > terceiro_numero:
  print(f'{primeiro_numero} é maior que {segundo_numero} e {terceiro_numero}')

elif segundo_numero > primeiro_numero and segundo_numero > terceiro_numero:
  print(f'{segundo_numero} é maior que {primeiro_numero} e {terceiro_numero}')

else:
  print(f'{terceiro_numero} é maior que {primeiro_numero} e {segundo_numero}')


Digite um número: 10
Outro número: 0
Mais um número: 900

900 é maior que 10 e 0


## Exercício 7

### Faça um Programa que leia três números e mostre o maior e o menor deles.

In [ ]:
numero01 = int(input('Digite 1° número: '))
numero02 = int(input('Digite 2° número: '))
numero03 = int(input('Digite 3° número: '))

maior_valor = numero01
if numero02 > numero01 and numero02 > numero03:
  maior_valor = numero02

elif numero03 > numero01 and numero03 > numero02:
  maior_valor = numero03

menor_valor = numero01
if numero02 < numero01 and numero02 < numero03:
  menor_valor = numero02

elif numero03 < numero01 and numero03 < numero02:
  menor_valor = numero03
print()
print(f'O Maior número é: {maior_valor}')
print(f'O Menor número é: {menor_valor}')


     Digite um número: 78
  Digite outro número: 79
Digite mais um número: 80

O Maior número é: 80
O Menor número é: 78


## Exercício 8

### Faça um programa que pergunte o preço de três produtos e informe qual produto você deve comprar, sabendo que a decisão é sempre pelo mais barato.

In [ ]:
placa_nvidia = float(input('Qual o valor da placa GeForce RTX 3070    R$ : '))
placa_amd = float(input('Qual o valor da placa Radeon RX 6600 XT   R$ : '))
placa_intel = float(input('Qual o valor da placa Intel Arc Alchemist R$ : '))
print()

placa_mais_cara = placa_nvidia
if placa_amd > placa_nvidia and placa_amd > placa_intel:
  placa_mais_cara = placa_amd
  print(f'O modelo mais caro é Radeon RX 6600 XT R$ {placa_mais_cara}')

elif placa_intel > placa_nvidia and placa_intel > placa_amd:
  placa_mais_cara = placa_intel
  print(f'O modelo mais caro é Intel Arc Alchemist R$ {placa_mais_cara}')

else:
  print(f'O modelo mais caro é GeForce RTX 3070 R$ {placa_mais_cara}')
print()

placa_mais_barata = placa_nvidia
if placa_amd < placa_nvidia and placa_amd < placa_intel:
  placa_mais_barata = placa_amd
  print(f'O modelo mais barato é Radeon RX 6600 XT R$ {placa_mais_barata}')

elif placa_intel < placa_nvidia and placa_intel < placa_amd:
  placa_mais_barata = placa_intel
  print(f'O modelo mais barato é Intel Arc Alchemist R$ {placa_mais_barata}')

else:
  print(f'O modelo mais barato é GeForce RTX 3070 R$ {placa_mais_barata}')


print()
print(f'Agora é com você, Boa Compra! ')


Qual o valor da placa GeForce RTX 3070    R$ : 566666
Qual o valor da placa Radeon RX 6600 XT   R$ : 5666666
Qual o valor da placa Intel Arc Alchemist R$ : 566667888

O modelo mais caro é Intel Arc Alchemist R$ 566667888.0

O modelo mais barato é GeForce RTX 3070 R$ 566666.0

Agora é com você, Boa Compra! 


## Exercício 9

### Faça um Programa que leia três números e mostre-os em ordem decrescente.

In [ ]:
primeiro_numero = float(input('Digite seu primeiro número: '))
segundo_numero = float(input('Digite seu segundo número: '))
terceiro_numero = float(input('Digite seu terceiro número: '))

if primeiro_numero > segundo_numero and segundo_numero > terceiro_numero:
  print( primeiro_numero, '>', segundo_numero, '>', terceiro_numero )

elif segundo_numero > primeiro_numero and primeiro_numero > terceiro_numero:
  print( segundo_numero, '>', primeiro_numero, '>', terceiro_numero )

elif segundo_numero > terceiro_numero and terceiro_numero > primeiro_numero:
  print( segundo_numero, '>', terceiro_numero, '>', primeiro_numero )

elif terceiro_numero > primeiro_numero and primeiro_numero > segundo_numero:
  print( terceiro_numero, '>', primeiro_numero, '>', segundo_numero )

elif terceiro_numero > segundo_numero and segundo_numero > primeiro_numero:
  print( terceiro_numero, '>', segundo_numero, '>', primeiro_numero )

else:
  print( primeiro_numero, '>', terceiro_numero, '>', segundo_numero )

Digite seu primeiro número: 3
Digite seu segundo número: 1
Digite seu terceiro número: 2
3.0 > 2.0 > 1.0


## Exercício 10

### Faça um Programa que pergunte em que turno você estuda. Peça para digitar M-matutino ou V-Vespertino ou N- Noturno. Imprima a mensagem "Bom Dia!", "Boa Tarde!" ou "Boa Noite!" ou "Valor Inválido!", conforme o caso.

In [ ]:
print(f'Qual Turno você estuda?')
turno = (input(f'Digite M - Matutino, V - Vespertino, N - Nortuno:  ')).upper()

if turno == 'M' or turno == 'Matutino':
  print('Bom Dia!')

elif turno == "V" or turno == 'Vespertino':
  print('Boa Tarde!')

elif turno == 'N' or turno == 'Noturno':
  print('Boa Noite!')

else:
  print(f'Valor Inválido')

Qual Turno você estuda?
Digite M - Matutino, V - Vespertino, N - Nortuno:  v
Boa Tarde!


## Exercício 11

### As Organizações Tabajara resolveram dar um aumento de salário aos seus colaboradores e lhe contraram para desenvolver o programa que calculará os reajustes.Faça um programa que recebe o salário de um colaborador e o reajuste segundo o seguinte critério, baseado no salário atual:
###salários até R'' 280,00 (incluindo)  aumento de 20%.

###salários entre R'' 280,00 e R'' 700,00 aumento de 15%

###salários entre R''  700,00 e R''  1500,00  aumento de 10%

###salários de R'' 1500,00 em diante  aumento de 5%
 
###Após o aumento ser realizado, informe na tela:
####o salário antes do reajuste
####o percentual de aumento aplicado
####o valor do aumento
####o novo salário, após o aumento.

In [ ]:
import math

salario = float(input('Quanto você recebe: R$ '))

aumento1 = salario  + (salario * 20 / 100)
aumento2 = salario  + (salario * 15 / 100)
aumento3 = salario  + (salario * 10 / 100)
aumento4 = salario  + (salario * 5 / 100)

if salario <= 280:
  print('Salário Atual é R$', salario)
  print('Seu Aumento é de 20%')
  print('Valor do Aumento: R$',aumento1 - salario )
  print('Seu novo Salário é R$',aumento1)

elif salario <= 700:
  print('Salário Atual é R$', salario)
  print('Seu Aumento é de 15%')
  print('Valor do Aumento: R$',aumento2 - salario )
  print('Seu novo Salário é R$',aumento2)

elif salario <= 1500:
  print('Salário Atual é R$', salario)
  print('Seu Aumento é de 10%')
  print('Valor do Aumento: R$',aumento3 - salario )
  print('Seu novo Salário é R$',aumento3)

elif salario > 1501:
  print('Salário Atual é R$', salario)
  print('Seu Aumento é de 5%')
  print('Valor do Aumento: R$',aumento4 - salario )
  print('Seu novo Salário é R$',aumento4)

else:
  print('Hello World')

Quanto você recebe: R$ 80
Salário Atual é R$ 80.0
Seu Aumento é de 20%
Valor do Aumento: R$ 16.0
Seu novo Salário é R$ 96.0


## Exercício 12

###### Faça um programa para o cálculo de uma folha de pagamento, sabendo que os descontos são do Imposto de Renda, que depende do salário bruto (conforme tabela abaixo) e 3% para o Sindicato e que o FGTS corresponde a 11% do Salário Bruto, mas não é descontado (é a empresa que deposita). O Salário Líquido corresponde ao Salário Bruto menos os descontos. O programa deverá pedir ao usuário o valor da sua hora e a quantidade de horas trabalhadas no mês.
Desconto do IR:

Salário Bruto até 900 (inclusive) - isento

Salário Bruto até 1500 (inclusive) - desconto de 5%

Salário Bruto até 2500 (inclusive) - desconto de 10%

Salário Bruto acima de 2500 - desconto de 20% 

Imprima na tela as informações, dispostas conforme o exemplo abaixo.

 No exemplo o valor da hora é 5 e a quantidade de hora é 220.

        Salário Bruto: (5 * 220)        : R$ 1100,00
        (-) IR (5%)                     : R$   55,00  
        (-) INSS ( 10%)                 : R$  110,00
        FGTS (11%)                      : R$  121,00
        Total de descontos              : R$  165,00
        Salário Liquido                 : R$  

In [ ]:
import math
hora_trab = float(input(f'Quanto você recebe por hora: '))
hora_mes = float(input(f'Quantas horas você trabalhou no Mês: '))

total_hr = math.floor(hora_trab * hora_mes)
sindicato = math.floor(total_hr * 3 / 100)

ir =  math.floor(total_hr + (total_hr * 5 / 100))
inss =  math.floor(total_hr + (total_hr * 10 / 100))
fgts =  math.floor(total_hr + (total_hr * 11 / 100))

print(f'Salário:', total_hr)

if total_hr <= 900:
  print(f'Desconto Sindicato 3%: R$', sindicato )
  print(f'Desconto IR: R$', ir - total_hr )
  print(f'Desconto INSS: R$', inss - total_hr)
  print(f'Desconto FGTS: R$', fgts - total_hr)
  print(f'Total de Desconto + (3%) Sindicato: R$', ( ir - total_hr) + (inss - total_hr) + sindicato)
  print(f'Salário Liquido: R$', total_hr - (( ir - total_hr) + (inss - total_hr) + sindicato))

elif total_hr <= 1500:
  print(f'Desconto Sindicato 3%: R$', sindicato )
  print(f'Desconto IR: R$', ir - total_hr )
  print(f'Desconto INSS: R$', inss - total_hr)
  print(f'Desconto FGTS: R$', fgts - total_hr)
  print(f'Total de Desconto + (3%) Sindicato: R$', ( ir - total_hr) + (inss - total_hr) + sindicato)
  print(f'Salário Liquido: R$', total_hr - (total_hr * 5 /100) - (( ir - total_hr) + (inss - total_hr) + sindicato))

elif total_hr <= 2500:
  print(f'Desconto Sindicato 3%: R$', sindicato )
  print(f'Desconto IR: R$', ir - total_hr )
  print(f'Desconto INSS: R$', inss - total_hr)
  print(f'Desconto FGTS: R$', fgts - total_hr)
  print(f'Total de Desconto + (3%) Sindicato: R$', ( ir - total_hr) + (inss - total_hr) + sindicato)
  print(f'Salário Liquido: R$', total_hr - (total_hr * 10 /100) - (( ir - total_hr) + (inss - total_hr) + sindicato))

elif total_hr >= 2500:
  print(f'Desconto Sindicato 3%: R$', sindicato )
  print(f'Desconto IR: R$', ir - total_hr )
  print(f'Desconto INSS: R$', inss - total_hr)
  print(f'Desconto FGTS: R$', fgts - total_hr)
  print(f'Total de Desconto + (3%) Sindicato: R$', ( ir - total_hr) + (inss - total_hr) + sindicato)
  print(f'Salário Liquido: R$', total_hr - (total_hr * 20 /100) - (( ir - total_hr) + (inss - total_hr) + sindicato))


Quanto você recebe por hora: 50
Quantas horas você trabalhou no Mês: 66
Salário: 3300
Desconto Sindicato 3%: R$ 99
Desconto IR: R$ 165
Desconto INSS: R$ 330
Desconto FGTS: R$ 363
Total de Desconto + (3%) Sindicato: R$ 594
Salário Liquido: R$ 2046.0


## Exercício 13

### Faça um Programa que leia um número e exiba o dia correspondente da semana. (1-Domingo, 2- Segunda, etc.), se digitar outro valor deve aparecer valor inválido.

In [ ]:
semana = float(input(f' Escolha um dia da semana: '))

if semana == 1:
  print(f'Domingo')

elif semana == 2:
  print(f'Segunda')

elif semana == 3:
  print(f'Terça')

elif semana == 4:
  print(f'Quarta')

elif semana == 5:
  print(f'Quinta')

elif semana == 6:
  print(f'Sexta')

elif semana == 7:
  print(f'Sábado')

 Escolha um dia da semana: 6
Sexta


## Exercício 14

### Faça um programa que lê as duas notas parciais obtidas por um aluno numa disciplina ao longo de um semestre, e calcule a sua média. A atribuição de conceitos obedece à tabela abaixo:
####Média de Aproveitamento  Conceito

Entre 9.0 e 10.0        A

Entre 7.5 e 9.0         B

Entre 6.0 e 7.5         C

Entre 4.0 e 6.0         D

Entre 4.0 e zero        E

O algoritmo deve mostrar na tela as notas, a média, o conceito correspondente e a mensagem “APROVADO” se o conceito for A, B ou C ou “REPROVADO” se o conceito for D ou E.

In [ ]:
disciplina_1 = float(input(f'Qual valor da nota:'))
disciplina_2 = float(input(f'Qual valor da nota:'))

media_dis = ( disciplina_1 + disciplina_2) / 2

if media_dis >= 9.0:
  print(f'Média do Aluno foi:', media_dis)
  print(f'Conceito: A')
  print(f'Aprovado')

elif media_dis >= 7.5:
  print(f'Média do Aluno foi:', media_dis)
  print(f'Conceito: B')
  print(f'Aprovado')

elif media_dis >= 6.0:
  print(f'Média do Aluno foi:', media_dis)
  print(f'Conceito: C')
  print(f'Aprovado')

elif media_dis >= 4.0:
  print(f'Média do Aluno foi:', media_dis)
  print(f'Conceito: D')
  print(f'Recuperação')

elif media_dis < 4 :
  print(f'Média do Aluno foi:', media_dis)
  print(f'Conceito: E')
  print(f'Reprovado')

Qual valor da nota:5
Qual valor da nota:5
Média do Aluno foi: 5.0
Conceito: D
Recuperação


## Exercício 15

### Faça um Programa que peça os 3 lados de um triângulo. O programa deverá informar se os valores podem ser um triângulo. Indique, caso os lados formem um triângulo, se o mesmo é: equilátero, isósceles ou escaleno.
Dicas:

Três lados formam um triângulo quando a soma de quaisquer dois lados for maior que o terceiro;

Triângulo Equilátero: três lados iguais;

Triângulo Isósceles: quaisquer dois lados iguais;

Triângulo Escaleno: três lados diferentes;

In [ ]:
lado1 = float(input(f'Digite um valor para o triângulo: '))
lado2 = float(input(f'Digite um valor para o triângulo: '))
lado3 = float(input(f'Digite um valor para o triângulo: '))

##triangulo
if lado1 > lado2 + lado3 or lado2 > lado1 + lado3 or lado3 > lado2 + lado1:
  print(f'Triângulo')

##Equilatero
elif lado1 == lado2 == lado3:
  print(f'Equilátero')

##Isósceles
elif lado1 == lado2 or lado1 == lado3 or lado2 == lado3:
  print(f'Isósceleso')

else:
  print(f'Escaleno')






Digite um valor para o triângulo: 4
Digite um valor para o triângulo: 7
Digite um valor para o triângulo: 9
Escaleno


## Exercício 16

### Faça um programa que calcule as raízes de uma equação do segundo grau, na forma ax2 + bx + c. O programa deverá pedir os valores de a, b e c e fazer as consistências, informando ao usuário nas seguintes situações:
Se o usuário informar o valor de A igual a zero, a equação não é do segundo grau e o programa não deve fazer pedir os demais valores, sendo encerrado;

Se o delta calculado for negativo, a equação não possui raizes reais. Informe ao usuário e encerre o programa;

Se o delta calculado for igual a zero a equação possui apenas uma raiz real; informe-a ao usuário;

Se o delta for positivo, a equação possui duas raiz reais; informe-as ao usuário;

In [ ]:
import math

print(f'Equação do 2° Grau: ax² + bx + c')
valor_a = float(input(f'Digite o valor de A: '))
valor_b = float(input(f'Digite o valor de B: '))
valor_c = float(input(f'Digite o valor de C: '))

delta = valor_b **2 - 4 * valor_a * valor_c
x1 = (-valor_b + delta**0.5) / 2.0 * valor_a
x2 = (-valor_b - delta**0.5) / 2.0 * valor_a

if valor_a == 0:
  if valor_a == 0:
   print(f'A Equação não é do 2° Grau! Finalizado a Equação')

  else:
   print( valor_a + x1**2 + valor_b * x1 + valor_c)
   print( valor_a + x2**2 + valor_b * x2 + valor_c)

elif delta < 0:
 print( valor_a + x1**2 + valor_b * x1 + valor_c)
 print( valor_a + x2**2 + valor_b * x2 + valor_c)
 print(f'A Equação não possui raizes reais')

elif delta == 0:
 print( valor_a + x1**2 + valor_b * x1 + valor_c)
 print( valor_a + x2**2 + valor_b * x2 + valor_c)
 print(f'A Equação possui um raiz real')

elif delta > 0:
 print( valor_a + x1**2 + valor_b * x1 + valor_c)
 print( valor_a + x2**2 + valor_b * x2 + valor_c)
 print(f'A Equação possui duas  raizes reais')


## valor de delta é: b ** 2 - 4 * a * c \\ valor de x é: -b + or - raiz quadrada de delta






Equação do 2° Grau: ax² + bx + c
Digite o valor de A: 5
Digite o valor de B: 5
Digite o valor de C: 5
(-365.00000000000017-433.0127018922193j)
(-365.00000000000006+433.0127018922194j)
A Equação não possui raizes reais


## Exercício 17

### Faça um Programa que peça um número correspondente a um determinado ano e em seguida informe se este ano é ou não bissexto.

In [ ]:
ano_bissexto = int(input(f'Digite um ano: '))

if ano_bissexto % 4 == 0 and ano_bissexto != 100 == 0 or ano_bissexto % 400 == 0:
  print(f'{ano_bissexto} é um ano Bissexto!')
else:
  print(f'{ano_bissexto} não é um ano Bissexto!')

Digite um ano: 1900
1900 não é um ano Bissexto!


## Exercício 18

### Faça um Programa que peça uma data no formato dd/mm/aaaa e determine se a mesma é uma data válida.

In [ ]:
dia = int(input(f'Escolha o dia: ')) 
mes = int(input(f'Escolha o mês: '))
ano = int(input(f'Escolha o ano: '))

if dia >= 32:
 if dia == 32:
   print(f'{dia} Dia Inválido')
 else:
   print(f'{dia} / {mes} / {ano}')

elif mes >= 13:
  if mes == 13:
    print(f'{mes} Mês Inválido')
  else:
    print(f'{dia} / {mes} / {ano}')

if mes == 2:
  if dia >= 30:
    print(f'{dia} não é Inválido para o Mês requerido!')
  else:
    print(f'{dia} / {mes} / {ano}')

print(f'Data:  {dia} / {mes} / {ano}')

if ano % 4 == 0 and ano % 100 != 0 or ano % 400 == 0:
  print(f'OBS: O ano {ano} é bissexto!')
else:
  print(f'OBS: O ano {ano} não é bissexto!')

Escolha o dia: 12
Escolha o mês: 12
Escolha o ano: 1212
Data:  12 / 12 / 1212
OBS: O ano 1212 é bissexto!


## Exercício 19

### Faça um Programa que leia um número inteiro menor que 1000 e imprima a quantidade de centenas, dezenas e unidades do mesmo.:
Observando os termos no plural a colocação do "e", da vírgula entre outros. Exemplo:
326 = 3 centenas, 2 dezenas e 6 unidades
12 = 1 dezena e 2 unidades Testar com: 326, 300, 100, 320, 310,305, 301, 101, 311, 111, 25, 20, 10, 21, 11, 1, 7 e 16

In [ ]:
numero_inteiro = int(input('Digite um Número: '))

centenas_str = dezenas_str = unidades_str = ''
centenas_int, numero_inteiro = divmod(numero_inteiro, 100)

partes_numericas = 0

if centenas_int == 1:
  centenas_str = f'1 centena'
  partes_numericas += 1
elif centenas_int > 1:
  centenas_str = f'{centenas_int} centenas'
  partes_numericas += 1

dezenas_int, numero_inteiro = divmod(numero_inteiro, 10)

if dezenas_int == 1:
  dezenas_str = f'1 dezena'
  partes_numericas += 1
elif dezenas_int > 1:
  dezenas_str = f'{dezenas_int} dezenas'
  partes_numericas += 1

if numero_inteiro == 1:
  unidades_str = f'1 unidade'
  partes_numericas += 1
elif numero_inteiro > 1:
  unidades_str = f'{numero_inteiro} unidades'
  partes_numericas += 1

if partes_numericas == 3:
  print(f'{centenas_str}, {dezenas_str} e {unidades_str} ')

if partes_numericas == 1:
  print(centenas_str + dezenas_str + unidades_str)

if partes_numericas == 2:
  if centenas_str != '':
    segunda_parte = (dezenas_str + unidades_str)  
    print(f'{centenas_str} e {segunda_parte}')
  else:
    print(f'{dezenas_str} e {unidades_str}')


Digite um Número: 709
7 centenas e 9 unidades


## Exercício 20

### Faça um Programa para leitura de três notas parciais de um aluno. O programa deve calcular a média alcançada por aluno e presentar:
A mensagem "Aprovado", se a média for maior ou igual a 7, com a respectiva média alcançada;

A mensagem "Reprovado", se a média for menor do que 7, com a respectiva média alcançada;

A mensagem "Aprovado com Distinção", se a média for igual a 10.

In [ ]:
primeira_nota = float(input('1° nota: '))
segunda_nota = float(input('2° nota: '))
terceira_nota = float(input('3° nota: '))

media_notas = (primeira_nota + segunda_nota + terceira_nota) // 3

print(f'Média final do Aluno: {media_notas}')

if media_notas == 10:
  print(f'Aprovado com Destinção')

elif media_notas > 6:
  print(f'Aprovado')

elif media_notas > 5:
  print(f'Recuperação')

elif media_notas <= 5:
  print(f'Reprovado')

1° nota: 5
2° nota: 5
3° nota: 5
Média final do Aluno: 5.0
Reprovado


## Exercício 21

### Faça um Programa para um caixa eletrônico. O programa deverá perguntar ao usuário a valor do saque e depois informar quantas notas de cada valor serão fornecidas. As notas disponíveis serão as de 1, 5, 10, 50 e 100 reais. O valor mínimo é de 10 reais e o máximo de 600 reais. O programa não deve se preocupar com a quantidade de notas existentes na máquina.
Exemplo 1: Para sacar a quantia de 256 reais, o programa fornece duas notas de 100, uma nota de 50, uma nota de 5 e uma nota de 1;

Exemplo 2: Para sacar a quantia de 399 reais, o programa fornece três notas de 100, uma nota de 50, quatro notas de 10, uma nota de 5 e quatro notas de 1.

In [ ]:
print('Caixa DevPro')
valor_saque = int(input('Valor a ser sacado: '))
nota_cem = nota_de_cinquenta = nota_de_dez = nota_de_cinco = nota_de_um = 0

nota_de_cem, valor_saque = divmod(valor_saque, 100)
if nota_de_cem < 1:
  nota_de_cem = ''
elif nota_de_cem == 1:
  print(f'{nota_de_cem} nota de 100')
else:
  print(f'{nota_de_cem} notas de 100')

nota_de_cinquenta, valor_saque = divmod(valor_saque, 50)
if nota_de_cinquenta < 1:
  nota_de_cinquenta = ''
elif nota_de_cinquenta == 1:
  print(f'{nota_de_cinquenta} nota de 50')
else:
  print(f'{nota_de_cinquenta} notas de 50')

nota_de_dez, valor_saque = divmod(valor_saque, 10)
if nota_de_dez < 1:
  nota_de_dez = ''
elif nota_de_dez == 1:
  print(f'{nota_de_dez} nota de 10')
else:
  print(f'{nota_de_dez} notas de 10')

nota_de_cinco, valor_saque = divmod(valor_saque, 5)
if nota_de_cinco < 1:
  nota_de_cinco = ''
elif nota_de_cinco == 1:
  print(f'{nota_de_cinco} nota de 5')
else:
  print(f'{nota_de_cinco} notas de 5')

nota_de_um, valor_saque = divmod(valor_saque, 1)
if nota_de_um < 1:
  nota_de_um = ''
elif nota_de_um == 1:
  print(f'{nota_de_um} nota de 1')
else:
  print(f'{nota_de_um} notas de 1')



Caixa DevPro
Valor a ser sacado: 80
1 nota de 50
3 notas de 10


## Exercício 22

### Faça um Programa que peça um número inteiro e determine se ele é par ou impar. 
Dica: utilize o operador módulo (resto da divisão).

In [ ]:
par_ou_impar = int(input('Digite um número: '))

if (par_ou_impar % 2) == 0:
  print(f'{par_ou_impar} é um número PAR')
else:
  print(f'{par_ou_impar} é um número IMPAR')

Digite um número: 69876
69876 é um número PAR


## Exercício 23

### Faça um Programa que peça um número e informe se o número é inteiro ou decimal. 
Dica: utilize uma função de arredondamento.

In [ ]:
import math
inteiro_ou_decimal = float(input('Digite um número: '))

if inteiro_ou_decimal == round(inteiro_ou_decimal):
  print(f'{int(inteiro_ou_decimal)} é um número inteiro')
else:
  print(f'{inteiro_ou_decimal} é um número decimal')

Digite um número: 4555565555.7
4555565555.7 é um número decimal


## Exercício 24

### Faça um Programa que leia 2 números e em seguida pergunte ao usuário qual operação ele deseja realizar. O resultado da operação deve ser acompanhado de uma frase que diga se o número é:

A) par ou ímpar;

B) positivo ou negativo;

C) inteiro ou decimal.

In [ ]:
primeiro_valor = float(input('Digite o 1° número: '))
segundo_valor = float(input('Digite o 2° número: '))

print('Escolha a operação que deseja realizar "S" para somar "SB" para subtrair "D" para dividir e "M" para multiplicar ')

operacao = input('Digite qual operação será realizada: ').upper()
resultado_valores = 0
par_impar = ''
inteiro_decimal = ''
positivo_negativo = ''

if operacao == 'S':  
  resultado_valores = primeiro_valor + segundo_valor
  print(f'Os valores serão somados')
if operacao == 'SB':
  resultado_valores = primeiro_valor - segundo_valor
  print(f'Os valores serão subtraídos')
if operacao == 'D':
  resultado_valores = primeiro_valor // segundo_valor
  print(f'Os valores serão divididos')
if operacao == 'M':
  resultado_valores = primeiro_valor * segundo_valor
  print(f'Os valores serão multiplicados')

if resultado_valores % 2 == 0:
  par_impar = 'PAR'
else:
  par_impar = 'IMPAR'

if resultado_valores > 0:
  positivo_negativo = 'Positivo'
else:
  positivo_negativo = 'Negativo'

if resultado_valores == round(resultado_valores):
  inteiro_decimal = 'Inteiro'
  resultado_valores = int(resultado_valores)
else:
  inteiro_decimal = 'Decimal'8

print(f'Resultado: {resultado_valores}')
print(f'Seu valor deu {positivo_negativo}')
print(f'{resultado_valores} é um número {par_impar}')
print(f'{resultado_valores} é um número {inteiro_decimal}')





Digite o 1° número: 67
Digite o 2° número: 7
Escolha a operação que deseja realizar "S" para somar "SB" para subtrair "D" para dividir e "M" para multiplicar 
Digite qual operação será realizada: s
Os valores serão somados
Resultado: 74
Seu valor deu Positivo
74 é um número PAR
74 é um número Inteiro


## Exercício 25

### Faça um programa que faça 5 perguntas para uma pessoa sobre um crime. As perguntas são:

"Telefonou para a vítima?"

"Esteve no local do crime?"

"Mora perto da vítima?"

"Devia para a vítima?"

"Já trabalhou com a vítima?"

 O programa deve no final emitir uma classificação sobre a participação da pessoa no crime. Se a pessoa responder positivamente a 2 questões ela deve ser classificada como "Suspeita", entre 3 e 4 como "Cúmplice" e 5 como "Assassino". Caso contrário, ele será classificado como "Inocente".

In [ ]:
print('<< Interrogatório >> ')
print()
pergunta_1 = input('Telefonou para vítima? ').upper()
pergunta_2 = input('Esteve no local do crime? ').upper()
pergunta_3 = input('Mora perto da vítima? ').upper()
pergunta_4 = input('Devia para a vítima? ').upper()
pergunta_5 = input('Já trabalhou com a vítima? ').upper()
print()

resposta = 0

if pergunta_1 == 'SIM' or pergunta_1 == 'S':
  resposta += 1

if pergunta_2 == 'SIM' or pergunta_2 == 'S':
  resposta += 1

if pergunta_3 == 'SIM' or pergunta_3 == 'S':
  resposta += 1

if pergunta_4 == 'SIM' or pergunta_4 == 'S':
  resposta += 1
  
if pergunta_5 == 'SIM' or pergunta_5 == 'S':
  resposta += 1

print(f'A investigação acabou:')
print()
if resposta >= 5:
  print(f'Você é o Assassino do crime')

elif resposta >= 3:
  print(f'Você foi Cúmplice no crime')

elif resposta >= 1:
  print(f'Você é Suspeito pelo crime ')

elif resposta == 0:
  print(f'Você é inocente no crime')


<< Interrogatório >> 

Telefonou para vítima? s
Esteve no local do crime? n
Mora perto da vítima? n
Devia para a vítima? n
Já trabalhou com a vítima? n

A investigação acabou:

Você é Suspeito pelo crime 


## Exercício 26

### Um posto está vendendo combustíveis com a seguinte tabela de descontos:

Álcool:

até 20 litros, desconto de 3% por litro

acima de 20 litros, desconto de 5% por litro

Gasolina:

até 20 litros, desconto de 4% por litro

acima de 20 litros, desconto de 6% por litro

 Escreva um algoritmo que leia o número de litros vendidos, o tipo de combustível (codificado da seguinte forma: A-álcool, G-gasolina) 
 
 Calcule e imprima o valor a ser pago pelo cliente sabendo-se que o preço do litro da gasolina é R$ 2,50 o preço do litro do álcool é RS 1,90.

In [ ]:
print('Posto quero barato')
print()
combustivel = input('Qual combustível você vai querer: A - álcool, G - gasolina: ').upper()
litros_combustivel = float(input('Quantos litros você vai querer: '))

alcool = 1.90
litros_por_alcool = litros_combustivel * alcool
alcool_abaixo_20 = litros_por_alcool * 0.03
desconto_alcool_abaixo_20 = litros_por_alcool - alcool_abaixo_20
alcool_acima_20 = litros_por_alcool * 0.05
desconto_alcool_acima_20 = litros_por_alcool - alcool_acima_20

gasolina = 2.50
litros_por_gasolina = litros_combustivel * gasolina
gasolina_abaixo_20 = litros_por_gasolina * 0.04
desconto_gasolina_abaixo_20 = litros_por_gasolina - gasolina_abaixo_20
gasolina_acima_20 = litros_por_gasolina * 0.06
desconto_gasolina_acima_20 = litros_por_gasolina - gasolina_acima_20

print()

if combustivel == "GASOLINA" or combustivel == 'G':
  if litros_combustivel > 20:
    print(f'Combustível: Gasolina')
    print(f'Quantidade: {litros_combustivel} litro(s)')
    print(f'Total: R$ {litros_por_gasolina}')
    print(f'Desconto: R$ {gasolina_acima_20}')
    print(f'Total: R$ {desconto_gasolina_acima_20}')
  else:
    print(f'Combustível: Gasolina')
    print(f'Quantidade: {litros_combustivel} litro(s)')
    print(f'Total: R$ {litros_por_gasolina}')
    print(f'Desconto: R$ {gasolina_abaixo_20}')
    print(f'Total: R$ {desconto_gasolina_abaixo_20}')

if combustivel == 'ALCOOL' or combustivel == 'ÁLCOOL' or combustivel == 'A':
  if litros_combustivel > 20:
    print(f'Combutível: Álcool')
    print(f'Quantidade: {litros_combustivel} litros(s)')
    print(f'Total: R$ {litros_por_alcool}')
    print(f'Desconto: R$ {alcool_acima_20}')
    print(f'Total: R$ {desconto_alcool_acima_20}')
  else:
    print(f'Combutível: Álcool')
    print(f'Quantidade: {litros_combustivel} litros(s)')
    print(f'Total: R$ {litros_por_alcool}')
    print(f'Desconto: R$ {alcool_abaixo_20}')
    print(f'Total: R$ {desconto_alcool_abaixo_20}')


Posto quero barato

Qual combustível você vai querer: A - álcool, G - gasolina: g
Quantos litros você vai querer: 21

Combustível: Gasolina
Quantidade: 21.0 litro(s)
Total: R$ 52.5
Desconto: R$ 3.15
Total: R$ 49.35


## Exercício 27

### Uma fruteira está vendendo frutas com a seguinte tabela de preços:

          Até 5 Kg           Acima de 5 Kg
Morango  RS 2,50 por Kg RS 2,20 por Kg

Maçã   RS 1,80 por Kg          RS 1,50 por Kg

Se o cliente comprar mais de 8 Kg em frutas ou o valor total da compra ultrapassar R$ 25,00, receberá ainda um desconto de 10% sobre este total. 

Escreva um algoritmo para ler a quantidade (em Kg) de morangos e a quantidade (em Kg) de maças adquiridas e escreva o valor a ser pago pelo cliente.

In [31]:
print("Xepa dos Dev's")

nome_fruta = input('Qual fruta você vai querer: Morango ou Maça: ').upper()
kilo_fruta = float(input('Quantos kilo você vai querer: '))
print()
#morango
morango_abaixo_5kg = 2.50
valor_morango_abaixo_5kg = morango_abaixo_5kg * kilo_fruta

morango_acima_5kg = 2.20
valor_morango_acima_5kg = morango_acima_5kg * kilo_fruta

morango_acima_8kg = valor_morango_acima_5kg * 1.10
valor_desconto_morango_8kg = morango_acima_8kg - valor_morango_acima_5kg 
valor_total_morango = valor_morango_acima_5kg - valor_desconto_morango_8kg

#maça
maça_abaixo_5kg = 1.80
valor_maça_abaixo_5kg = kilo_fruta * maça_abaixo_5kg

maça_acima_5kg = 1.50
valor_maça_acima_5kg = kilo_fruta * maça_acima_5kg

maça_acima_8kg = valor_maça_acima_5kg * 1.10
valor_desconto_maça_8kg = maça_acima_8kg - valor_maça_acima_5kg
valor_total_maça = valor_maça_acima_5kg - valor_desconto_maça_8kg

if nome_fruta == 'MORANGO':
  if kilo_fruta < 5:
    print(f'Fruta: Morango')
    print(f'Valor: R$ {valor_morango_abaixo_5kg}')
  elif kilo_fruta < 8:
    print(f'Fruta: Morango')
    print(f'Valor: R$ {valor_morango_acima_5kg}')
  elif kilo_fruta >8:
    print(f'Fruta: Morango')
    print(f'Valor: R$ {valor_morango_acima_5kg:.2f}')
    print(f'Desconto: R$ {valor_desconto_morango_8kg:.2f}')
    print(f'Valor Final: R$ {valor_total_morango:.2f}')

if nome_fruta == 'MAÇA':
  if kilo_fruta < 5:
    print(f'Fruta: Maça')
    print(f'Valor: R$ {valor_maça_abaixo_5kg}')
  elif kilo_fruta < 8:
    print(f'Fruta: Maça')
    print(f'Valor: R$ {valor_maça_acima_5kg}')
  elif kilo_fruta >= 8:
    print(f'Fruta: Maça')
    print(f'Valor: R$ {maça_acima_8kg:.2f}')
    print(f'Desconto: R$ {valor_desconto_maça_8kg:.2f}')
    print(f'Valor Final: R$ {valor_total_maça:.2f}')

# para ultrapassar o valor de R$ 25,00 o kilo da fruta já bateu mais de 8, considerando assim o desconto de 10% 

Xepa dos Dev's
Qual fruta você vai querer: Morango ou Maça: maça
Quantos kilo você vai querer: 5

Fruta: Maça
Valor: R$ 7.5


## Exercício 28

### O Hipermercado Tabajara está com uma promoção de carnes que é imperdível. Confira:
                      Até 5 Kg           Acima de 5 Kg

File Duplo      RS 5,80 por Kg          RS 4,90 por Kg

Alcatra         RS 6,80 por Kg          RS 5,90 por Kg

Picanha         RS 7,80 por Kg          RS 6,90 por Kg

Para atender a todos os clientes, cada cliente poderá levar apenas um dos tipos de carne da promoção, porém não há limites para a quantidade de carne por cliente. 

Se compra for feita no cartão Tabajara o cliente receberá ainda um desconto de 5% sobre o total da compra. 

Escreva um programa que peça o tipo e a quantidade de carne comprada pelo usuário e gere um cupom fiscal, contendo as informações da compra: tipo e quantidade de carne, preço total, tipo de pagamento, valor do desconto e valor a pagar.

In [43]:
print('Hipermercado Tabajara DevPro')
print()

carnes = input('Qual carne na promoção você vai querer: Filé Duplo, Alcatra ou Picanha: ').upper()
kilos_carne = float(input('Quantos Kilos: '))

print()
forma_de_pagamento = input('Qual a forma de Pagamento? Dinheiro, Cartão Visa ou Mastercard, cartão Tabajara: ').upper()

escolha_debito_credito = ''

# FORMA DE PAGAMENTO NO CARTÃO
if forma_de_pagamento == 'CARTÃO VISA' or forma_de_pagamento == 'VISA' or forma_de_pagamento == 'MASTER' or forma_de_pagamento == 'CARTÃO MASTER':
  escolha_debito_credito = input(f'Debito ou crédito? ').upper()
  if escolha_debito_credito == 'DEBITO':
   escolha_debito_credito = 'DEBITO'
  else:
    escolha_debito_credito = 'CRÉDITO'
elif forma_de_pagamento == 'DINHEIRO' or forma_de_pagamento == 'DN':
  escolha_debito_credito = 'DINHEIRO'
else:
  escolha_debito_credito = 'TABAJARA'

# FILE CARNE ABAIXO DE 5KG
file_abaixo_5kg = 5.80
valor_file_abaixo_5kg = kilos_carne * file_abaixo_5kg
# PAGAMENTO NO CARTÃO TABAJARA ABAIXO DE 5KG
valor_file_abaixo5kg_ct = valor_file_abaixo_5kg * 1.05
desconto_file_abaixo5kg_ct = valor_file_abaixo5kg_ct - valor_file_abaixo_5kg
valor_final_file_abaixo5kg_ct = valor_file_abaixo_5kg - desconto_file_abaixo5kg_ct
# FILE CARNE ACIMA DE 5 KG
file_acima_5kg  = 4.90
valor_file_acima_5kg = kilos_carne * file_acima_5kg
# PAGAMENTO CARTÃO TABAJARA ACIMA DE 5 KG
valor_file_acima5kg_ct = valor_file_acima_5kg * 1.05
desconto_file_acima5kg_ct = valor_file_acima5kg_ct - valor_file_acima_5kg
valor_final_file_acima5kg_ct = valor_file_acima_5kg - desconto_file_acima5kg_ct

# ALCATRA CARNE ABAIXO DE 5KG
alcatra_abaixo_5kg = 5.80
valor_alcatra_abaixo_5kg = kilos_carne * alcatra_abaixo_5kg
# PAGAMENTO NO CARTÃO TABAJARA ABAIXO DE 5KG
valor_alcatra_abaixo5kg_ct = valor_alcatra_abaixo_5kg * 1.05
desconto_alcatra_abaixo5kg_ct = valor_alcatra_abaixo5kg_ct - valor_alcatra_abaixo_5kg
valor_final_alcatra_abaixo5kg_ct = valor_alcatra_abaixo_5kg - desconto_alcatra_abaixo5kg_ct
# ALCATRA CARNE ACIMA DE 5 KG
alcatra_acima_5kg  = 4.90
valor_alcatra_acima_5kg = kilos_carne * alcatra_acima_5kg
# PAGAMENTO CARTÃO TABAJARA ACIMA DE 5 KG
valor_alcatra_acima5kg_ct = valor_alcatra_acima_5kg * 1.05
desconto_alcatra_acima5kg_ct = valor_alcatra_acima5kg_ct - valor_alcatra_acima_5kg
valor_final_alcatra_acima5kg_ct = valor_alcatra_acima_5kg - desconto_alcatra_acima5kg_ct

# PICANHA CARNE ABAIXO DE 5KG
picanha_abaixo_5kg = 5.80
valor_picanha_abaixo_5kg = kilos_carne * picanha_abaixo_5kg
# PAGAMENTO NO CARTÃO TABAJARA ABAIXO DE 5KG
valor_picanha_abaixo5kg_ct = valor_picanha_abaixo_5kg * 1.05
desconto_picanha_abaixo5kg_ct = valor_picanha_abaixo5kg_ct - valor_picanha_abaixo_5kg
valor_final_picanha_abaixo5kg_ct = valor_picanha_abaixo_5kg - desconto_picanha_abaixo5kg_ct
# PICANHA CARNE ACIMA DE 5 KG
picanha_acima_5kg  = 4.90
valor_picanha_acima_5kg = kilos_carne * picanha_acima_5kg
# PAGAMENTO CARTÃO TABAJARA ACIMA DE 5 KG
valor_picanha_acima5kg_ct = valor_picanha_acima_5kg * 1.05
desconto_picanha_acima5kg_ct = valor_picanha_acima5kg_ct - valor_picanha_acima_5kg
valor_final_picanha_acima5kg_ct = valor_picanha_acima_5kg - desconto_picanha_acima5kg_ct

print()
if carnes == 'FILE DUPLO' or carnes == 'FILÉ DUPLO':
  if forma_de_pagamento == 'CARTÃO TABAJARA' or forma_de_pagamento == 'TABAJARA':
    if kilos_carne < 5:
      print(f'Carne: Filé Duplo')
      print(f'Quant. Kilo(s): {kilos_carne}')
      print(f'Valor Total: R$ {valor_file_abaixo_5kg:.2f}')
      print(f'forma de Pagamento: Cartão {escolha_debito_credito} Internacional')
      print(f'Desconto: R$ {desconto_file_abaixo5kg_ct:.2f}')
      print(f'Valor Total: R$ {valor_final_file_abaixo5kg_ct:.2f}')
    elif kilos_carne >= 5:
      print(f'Carne: Filé Duplo')
      print(f'Quant. Kilo(s): {kilos_carne}')
      print(f'Valor Total: R$ {valor_file_acima_5kg:.2f}')
      print(f'forma de Pagamento: Cartão {escolha_debito_credito} Internacional')
      print(f'Desconto: R$ {desconto_file_acima5kg_ct:.2f}')
      print(f'Valor Total: R$ {valor_final_file_acima5kg_ct:.2f}')
  elif forma_de_pagamento == 'CARTÃO VISA' or forma_de_pagamento == 'VISA' or forma_de_pagamento == 'MASTER' or forma_de_pagamento == 'CARTÃO MASTER':
    if kilos_carne < 5:
      print(f'Desconto: Somente nos Cartões Tabajara e Tabajara Internacional')
      print(f'Carne: Filé Duplo')
      print(f'Quant. Kilo(s): {kilos_carne}')
      print(f'Valor Total: R$ {valor_file_abaixo_5kg:.2f}')
      print(f'forma de Pagamento: Cartão de {escolha_debito_credito}')      
      print(f'Total a Pagar: R$ {valor_file_abaixo_5kg:.2f}')
    elif kilos_carne >= 5:
      print(f'Desconto: Somente nos Cartões Tabajara e Tabajara Internacional')
      print(f'Carne: Filé Duplo')
      print(f'Quant. Kilo(s): {kilos_carne}')
      print(f'Valor Total: R$ {valor_file_acima_5kg:.2f}')
      print(f'forma de Pagamento: Cartão de {escolha_debito_credito}')      
      print(f'Total a Pagar: R$ {valor_file_acima_5kg:.2f}')
  elif forma_de_pagamento == 'DINHEIRO' or forma_de_pagamento == 'DN':
    if kilos_carne < 5:
      print(f'Desconto: Somente nos Cartões Tabajara e Tabajara Internacional')
      print(f'Carne: Filé Duplo')
      print(f'Quant. Kilo(s): {kilos_carne}')
      print(f'Valor Total: R$ {valor_file_abaixo_5kg:.2f}')
      print(f'forma de Pagamento: {escolha_debito_credito}')      
      print(f'Total a Pagar: R$ {valor_file_abaixo_5kg:.2f}')
    elif kilos_carne >= 5:
      print(f'Desconto: Somente nos Cartões Tabajara e Tabajara Internacional')
      print(f'Carne: Filé Duplo')
      print(f'Quant. Kilo(s): {kilos_carne}')
      print(f'Valor Total: R$ {valor_file_acima_5kg:.2f}')
      print(f'forma de Pagamento: {escolha_debito_credito}')      
      print(f'Total a Pagar: R$ {valor_file_acima_5kg:.2f}')

if carnes == 'ALCATRA':
  if forma_de_pagamento == 'CARTÃO TABAJARA' or forma_de_pagamento == 'TABAJARA':
    if kilos_carne < 5:
      print(f'Carne: Alcatra')
      print(f'Quant. Kilo(s): {kilos_carne}')
      print(f'Valor Total: R$ {valor_alcatra_abaixo_5kg:.2f}')
      print(f'forma de Pagamento: Cartão {escolha_debito_credito} Internacional')
      print(f'Desconto: R$ {desconto_alcatra_abaixo5kg_ct:.2f}')
      print(f'Valor Total: R$ {valor_final_alcatra_abaixo5kg_ct:.2f}')
    elif kilos_carne >= 5:
      print(f'Carne: Alcatra')
      print(f'Quant. Kilo(s): {kilos_carne}')
      print(f'Valor Total: R$ {valor_alcatra_acima_5kg:.2f}')
      print(f'forma de Pagamento: Cartão {escolha_debito_credito} Internacional')
      print(f'Desconto: R$ {desconto_alcatra_acima5kg_ct:.2f}')
      print(f'Valor Total: R$ {valor_final_alcatra_acima5kg_ct:.2f}')
  elif forma_de_pagamento == 'CARTÃO VISA' or forma_de_pagamento == 'VISA' or forma_de_pagamento == 'MASTER' or forma_de_pagamento == 'CARTÃO MASTER':
    if kilos_carne < 5:
      print(f'Desconto: Somente nos Cartões Tabajara e Tabajara Internacional')
      print(f'Carne: Alcatra')
      print(f'Quant. Kilo(s): {kilos_carne}')
      print(f'Valor Total: R$ {valor_alcatra_abaixo_5kg:.2f}')
      print(f'forma de Pagamento: Cartão de {escolha_debito_credito}')      
      print(f'Total a Pagar: R$ {valor_alcatra_abaixo_5kg:.2f}')
    elif kilos_carne >= 5:
      print(f'Desconto: Somente nos Cartões Tabajara e Tabajara Internacional')
      print(f'Carne: Alcatra')
      print(f'Quant. Kilo(s): {kilos_carne}')
      print(f'Valor Total: R$ {valor_alcatra_acima_5kg:.2f}')
      print(f'forma de Pagamento: Cartão de {escolha_debito_credito}')      
      print(f'Total a Pagar: R$ {valor_alcatra_acima_5kg:.2f}')
  elif forma_de_pagamento == 'DINHEIRO' or forma_de_pagamento == 'DN':
    if kilos_carne < 5:
      print(f'Desconto: Somente nos Cartões Tabajara e Tabajara Internacional')
      print(f'Carne: Alcatra')
      print(f'Quant. Kilo(s): {kilos_carne}')
      print(f'Valor Total: R$ {valor_alcatra_abaixo_5kg:.2f}')
      print(f'forma de Pagamento: {escolha_debito_credito}')      
      print(f'Total a Pagar: R$ {valor_alcatra_abaixo_5kg:.2f}')
    elif kilos_carne >= 5:
      print(f'Desconto: Somente nos Cartões Tabajara e Tabajara Internacional')
      print(f'Carne: Alcatra')
      print(f'Quant. Kilo(s): {kilos_carne}')
      print(f'Valor Total: R$ {valor_alcatra_acima_5kg:.2f}')
      print(f'forma de Pagamento: {escolha_debito_credito}')      
      print(f'Total a Pagar: R$ {valor_alcatra_acima_5kg:.2f}')

if carnes == 'PICANHA':
  if forma_de_pagamento == 'CARTÃO TABAJARA' or forma_de_pagamento == 'TABAJARA':
    if kilos_carne < 5:
      print(f'Carne: Picanha')
      print(f'Quant. Kilo(s): {kilos_carne}')
      print(f'Valor Total: R$ {valor_picanha_abaixo_5kg:.2f}')
      print(f'forma de Pagamento: Cartão {escolha_debito_credito} Internacional')
      print(f'Desconto: R$ {desconto_picanha_abaixo5kg_ct:.2f}')
      print(f'Valor Total: R$ {valor_final_picanha_abaixo5kg_ct:.2f}')
    elif kilos_carne >= 5:
      print(f'Carne: Picanha')
      print(f'Quant. Kilo(s): {kilos_carne}')
      print(f'Valor Total: R$ {valor_picanha_acima_5kg:.2f}')
      print(f'forma de Pagamento: Cartão {escolha_debito_credito} Internacional')
      print(f'Desconto: R$ {desconto_picanha_acima5kg_ct:.2f}')
      print(f'Valor Total: R$ {valor_final_picanha_acima5kg_ct:.2f}')
  elif forma_de_pagamento == 'CARTÃO VISA' or forma_de_pagamento == 'VISA' or forma_de_pagamento == 'MASTER' or forma_de_pagamento == 'CARTÃO MASTER':
    if kilos_carne < 5:
      print(f'Desconto: Somente nos Cartões Tabajara e Tabajara Internacional')
      print(f'Carne: Picanha')
      print(f'Quant. Kilo(s): {kilos_carne}')
      print(f'Valor Total: R$ {valor_picanha_abaixo_5kg:.2f}')
      print(f'forma de Pagamento: Cartão de {escolha_debito_credito}')      
      print(f'Total a Pagar: R$ {valor_picanha_abaixo_5kg:.2f}')
    elif kilos_carne >= 5:
      print(f'Desconto: Somente nos Cartões Tabajara e Tabajara Internacional')
      print(f'Carne: Picanha')
      print(f'Quant. Kilo(s): {kilos_carne}')
      print(f'Valor Total: R$ {valor_picanha_acima_5kg:.2f}')
      print(f'forma de Pagamento: Cartão de {escolha_debito_credito}')      
      print(f'Total a Pagar: R$ {valor_picanha_acima_5kg:.2f}')
  elif forma_de_pagamento == 'DINHEIRO' or forma_de_pagamento == 'DN':
    if kilos_carne < 5:
      print(f'Desconto: Somente nos Cartões Tabajara e Tabajara Internacional')
      print(f'Carne: Picanha')
      print(f'Quant. Kilo(s): {kilos_carne}')
      print(f'Valor Total: R$ {valor_picanha_abaixo_5kg:.2f}')
      print(f'forma de Pagamento: {escolha_debito_credito}')      
      print(f'Total a Pagar: R$ {valor_picanha_abaixo_5kg:.2f}')
    elif kilos_carne >= 5:
      print(f'Desconto: Somente nos Cartões Tabajara e Tabajara Internacional')
      print(f'Carne: Picanha')
      print(f'Quant. Kilo(s): {kilos_carne}')
      print(f'Valor Total: R$ {valor_picanha_acima_5kg:.2f}')
      print(f'forma de Pagamento: {escolha_debito_credito}')      
      print(f'Total a Pagar: R$ {valor_picanha_acima_5kg:.2f}')

Hipermercado Tabajara DevPro

Qual carne na promoção você vai querer: Filé Duplo, Alcatra ou Picanha: picanha
Quantos Kilos: 8

Qual a forma de Pagamento? Dinheiro, Cartão Visa ou Mastercard, cartão Tabajara: dinheiro

Desconto: Somente nos Cartões Tabajara e Tabajara Internacional
Carne: Picanha
Quant. Kilo(s): 8.0
Valor Total: R$ 39.20
forma de Pagamento: DINHEIRO
Total a Pagar: R$ 39.20
